src/documents/EPUB3Resources.md

```diff
@@ -3,34 +3,57 @@ title: EPUB 3 Resources
+   GNU GPL v3.
+10. [MathJax.org](http://www.mathjax.org/resources/epub-readers/) (Date unknown) - Resource has catered EPub 3
+    information to show where MathLM is supported
+11. [EPUBTest Results](http://www.epubtest.org/results/?view=detail) - Development is sponsored by the Daisy Consortium
+    and the Book Industry Study Group. Copyright International Digital Publishing Forum.
+12. [Tablets, Ereaders Lead to More Reading (October 17,
+    2013)](http://www.emarketer.com/Article/Tablets-Ereaders-Lead-More-Book-Reading/1010307) - Showing statistics in the
+    US that show the increase in reading due to the increase in digital ebook devices
+13. Accessible EReaders Sourcing An Accessible EReader (May 20, 2014) by Tim Lohman -
+    [http://www.accessiq.org/news/news/2014/05/accessible-e-readers-sourcing-an-accessible-e-reader](http://www.accessiq.org/news/news/2014/05/accessible-e-readers-sourcing-an-accessible-e-reader)
+    – review of accessible ereaders
+14. Accessibility Screening Methodology Guidelines and Checklist by Daisy Consortium -
+    [http://www.daisy.org/accessibility-screening-methodology-guidelines-and-checklist.html](http://www.daisy.org/accessibility-screening-methodology-guidelines-and-checklist.html)
+    – a checklist to help evaluate the accessibility features of a reader system
+15. Poet Image Description Tool -
+    [http://diagramcenter.org/development/poet.html](http://diagramcenter.org/development/poet.html). A web-based tool
+    for creating and providing guidelines to writing image descriptions for images in existing DAISY and EPUB books.
+16. MathML Cloud Tool * [http://mathmlcloud.org](http://mathmlcloud.org/). A math conversion tool that creates both
+    images and alternative descriptions of math equations on the fly.
 
 ## Topics:
 
-* [Inclusive EPUB3](/InclusiveEPUB3.html)
-* [Semantic Markup in EPUB 3](/SemanticMarkupInEPUB3.html)
-* [WAI-ARIA](/WAI-ARIA.html)
-* [Visual Styles](/VisualStyles.html)
-* [Graphics and Multimedia](/GraphicsAndMultimedia.html)
-* [Scripting in EPUB 3](/ScriptingInEPUB3.html)
-* [Mathematics](/Mathematics.html)
-* [Narrations and Text to Speech](/NarrationsAndTextToSpeech.html)
-* [Accessibility Metadata](/AccessibilityMetadata.html)
-* [Testing and Validation](/TestingAndValidation.html)
-* [Case Study Exemplar](/CaseStudyExemplar.html)
-* [EPUB 3 Resources](/EPUB3Resources.html)
+- [Inclusive EPUB3](/InclusiveEPUB3.html)
+- [Semantic Markup in EPUB 3](/SemanticMarkupInEPUB3.html)
+- [WAI-ARIA](/WAI-ARIA.html)
+- [Visual Styles](/VisualStyles.html)
+- [Graphics and Multimedia](/GraphicsAndMultimedia.html)
+- [Scripting in EPUB 3](/ScriptingInEPUB3.html)
+- [Mathematics](/Mathematics.html)
+- [Narrations and Text to Speech](/NarrationsAndTextToSpeech.html)
+- [Accessibility Metadata](/AccessibilityMetadata.html)
+- [Testing and Validation](/TestingAndValidation.html)
+- [Case Study Exemplar](/CaseStudyExemplar.html)
+- [EPUB 3 Resources](/EPUB3Resources.html)
```

Comment:
Why the change from "*" to "-"? It seems that we use "*" in other lists throughout.

Reply:
For some reason the linter was flagging this one. I'm not entirely sure.

Reply:
Fun with linting:

1. The default bullet style for ordered lists is `consistent`, aka either `*` or `-` as long as you are consistent within the same file.
2. The ordered list of resources at the beginning of this file uses hyphens to split up content, e.g.:
    > EPUB3 Overview - www.idpf.org/epub/30/spec/epub30-overview.html - answers basic questions about what is EPUB, what it is capable of, and how it can be used.
3. Wrapping to 120 columns ended up placing some of those hyphens at the start of a line, making MarkdownLint (and Markdown) interpret them as nested ordered lists.
4. Now we are failing the consistency rule by using `*` for bullets on the real unordered list at the bottom of the file.

I resolved this by using em dashes instead of hyphens above.